text/0229-otel-distributions.md

```diff
@@ -0,0 +1,137 @@
+# OpenTelemetry Collector Distributions
+
+Design and specification of OpenTelemetry Collector distributions.
+
+## Motivation
+
+The OpenTelemetry Collector is currently distributed by the OpenTelemetry project through two distributions, core and contrib.
+
+In numerous discussions with the community, maintainers and a few SIG meetings, it has emerged that those distributions
+emerged organically. We discuss them in detail below.
+
+This OTEP aims to clarify what distributions the OpenTelemetry project should distribute. The OTEP will elicit all requirements
+and dependencies attached to the current distributions, discuss the tooling in place, and propose a way forward.
+
+This OTEP also aims to become the center of discussions for the requirements and needs of the community, rather than
+seeing this issue being rehashed.
+
+### Core distribution
+The core distribution is defined in the [opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol) repository.
+
+The core distribution maps to well established technologies that predate OpenTelemetry, such as Jaeger, Zipkin, Prometheus and OpenCensus.
```

Comment:
```suggestion
The core distribution maps to well-established open source observability technologies that predate OpenTelemetry, such as Jaeger, Zipkin, Prometheus and OpenCensus.
```

I think there's slightly more nuance here than "before OTel".

Alternately, it could be prhased as such:

```suggestion
The core distribution exists to support OpenTelemetry's original mission to supersede OpenTracing and OpenCensus.  As such, it  includes components supporting signals and technologies supported by those systems.
```